src/actions/assetsActions.js

```diff
@@ -615,7 +616,7 @@ export const fetchAssetsBalancesAction = () => {
     dispatch({ type: SET_FETCHING_ASSETS_BALANCES, payload: true });
 
     dispatch(fetchSupportedAssetsAction());
-    dispatch(fetchPopularAssetsAction());
+    if (!popularAssets?.[CHAIN.POLYGON]) dispatch(fetchPopularAssetsAction());
```

Comment:
Was there an issue with Polygon popular assets?

Reply:
No no there is not an issue. I put a condition that if popular assets are available then don’t call this method again. we have not any other way to find assets available or accept this one.